WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/AddLocalMediaToPostUseCase.kt

```diff
@@ -50,6 +54,45 @@ class AddLocalMediaToPostUseCase @Inject constructor(
         editorMediaListener: EditorMediaListener,
         doUploadAfterAdding: Boolean = true,
         trackEvent: Boolean = true
+    ): Boolean {
+        var result = false
+
+        if (site.hasFreePlan) {
+            uriList.map {
+                result = if (mediaUtilsWrapper.isVideo(it) &&
```

Comment:
I encountered an issue while testing this PR where I could not upload a video, even when it was shorter than 5 minutes.

While debugging, I noticed that [this method makes assumptions about the content](https://github.com/wordpress-mobile/WordPress-Utils-Android/blob/1ed207c03d2242b6fc3d74f9e388e9163cbc82a6/WordPressUtils/src/main/java/org/wordpress/android/util/MediaUtils.java#L77) of the resource based on the name (in particular what the URL ends with). In this case, the URL (or URI) is `"content://media/external/video/media/594"`, and thus `isVideo` incorrectly returns `false`.

Though, even if the method worked as intended, I wonder whether we should be checking this here? My concern is that even if that method was working as intended, we'd be showing the wrong warning message; i.e. if a non-video file was detected, we'd warn about the duration limit.

Reply:
updated to check mime type beside uri

Reply:
Nice, I tested again, and it is working now.